distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+==== Certificate Management
+
+${branding} uses certificates in two ways:
+
+["lowerroman"]
+. Transmitting and receiving encrypted messages.
+. Authenticating an incoming user request.
+
+For more certificate configuration options, see <<_configuring_${ddf-security}, Configuring ${ddf-security}>>.
+
+===== Keystores
+
+To manage certificates and keys, ${branding} includes a default keystore.
+
+.Default Keystore Files
+[cols="5m" options="header"]
+|===
+
+|Host
+|Keystore
+|Truststore
+|Configuration Location
+|Usage
+
+|localhost
+|serverKeystore.jks
+|serverTruststore.jks
+|etc/org.ops4j.pax.web.cfg
+
+etc/system.properties
+
+etc/ws-security/server/encryption.properties
+
+etc/ws-security/server/signature.properties
+
+etc/ws-security/issuer/encryption.properties
+
+etc/ws-security/issuer/signature.properties
+
+etc/system.properties
+|Used to secure (SSL) all of the endpoints for ${branding}, to perform outgoing SSL requests, and sign STS SAML assertions.
+This also includes the ${admin-console} and any other web service that is hosted by ${branding}.
+
+|===
+
+[NOTE]
+====
+The following information was sourced from https://www.racf.bnl.gov/terapaths/software/the-terapaths-api/example-java-client/java-client/setting-up-keystores-with-jetty-and-keytool.
+====
+
+====== Creating a Client Keystore
+
+[WARNING]
+====
+This walk-through details how to use a PKCS12 store. This is the most popular format used when exporting from a web browser.
```

Comment:
Delete line 56 and replace it with:

ECA certificates are personal credentials used by the DoD to ensure the identity of those accessing DoD information systems. An ECA certificate can be exported from a web browser and imported into the ${branding} keystore. This steps provides the steps to export an ECA certificate from Internet Explorer.